File: 16-digitalclock.py
import time 

my_time=int(input('Enter time in seconds: '))
for x in range(my_time,0,-1):
  seconds = int(x%60)
  minutes=int(x/60)%60
  hours =int(x/3600) %24
  days = int(x/(24*3600))%30
  time.sleep(1)
  print(f'{hours:02}:{minutes:02}:{seconds:02}')

print('Times Up!!')


given_time = int(input('Enter the time in second: '))
current_time =0
for x in range(given_time+1):
  minutes=int(x/60)%60
  seconds = int(x%60)
  hours = int(x/3600)
  print(f'{hours:02}:{minutes:02}:{seconds:02}')
  time.sleep(1)

File: 16-digitalclock.js
let sec = 3665;

let timer = setInterval(() => {
  if (sec < 0) {
    clearInterval(timer);
    console.log("times Up!!");
  } else {
    sec--;
    let seconds = sec % 60;
    let minutes = Math.floor((sec / 60) % 60);
    let hours = Math.floor(sec / 3600);
    console.log(
      `${hours.toString().padStart(2, "0")}:${minutes
        .toString()
        .padStart(2, "0")}:${seconds.toString().padStart(2, "0")}`
    );
  }
}, 1000);
